hack/verify_boilerplate.py

```diff
@@ -135,6 +133,13 @@ def file_passes(filename, refs, regexs):  # pylint: disable=too-many-locals
     # trim our file to the same number of lines as the reference file
     data = data[:len(ref)]
 
+    # remove the 'YEAR' placeholder from the ref if the file is generated
+    if is_generated(file_data):
```

Comment:
looks like we are updating very specific files ... from this list:
https://github.com/kubernetes/repo-infra/blob/master/hack/verify_boilerplate.py#L76-L93

Also looks like the year is not needed in CNCF guidance:
https://github.com/cncf/foundation/blob/master/copyright-notices.md

So for this specific set of generated files, +1 to drop the year. Let's just not do this for any other files.

thanks!

Reply:
Should we instead display an error if YEAR is used in the copyright header for generated files (similar to k/k)? This would help ensure consistency across generated files.

https://github.com/kubernetes/kubernetes/blob/046f93b1133dadc6358b656d4000006b0953499b/hack/boilerplate/boilerplate.py#L124-L126

Reply:
@nikhita yes

I fear that we will break existing licence files if we merge a change like this.

Reply:
though repo-infra is not immediately upgraded anywhere and it shouldn't block go upgrades anymore ™️  now that k/k doesn't use bazel

Reply:
Changed it to return an error if it contains the `YEAR` placeholder and is generated.